Documentation/guides/DotNet5.md

```diff
@@ -0,0 +1,30 @@
+# .NET 5 and Xamarin.Android
+
+_NOTE: this document is very likely to change, as the requirements for
+.NET 5 are better understood._
+
+A .NET 5 project for Xamarin.Android will look something like:
+
+```xml
+<Project Sdk="Xamarin.Android.Sdk/0.0.1">
+  <PropertyGroup>
+    <TargetFramework>netcoreapp5.0</TargetFramework>
```

Comment:
As per the [**Target Framework Names in .NET 5** spec](https://github.com/terrajobst/designs/blob/25f4f0c7f710fb8205bb28147bb91b0471cb4840/accepted/2020/net5/net5.md), this will *probably* be `net5.0-android`, not `netcoreapp5.0`.

Reply:
I updated to match the spec, it also should have `<Project Sdk="Microsoft.NET.Sdk" />`.

Our current test projects have to import `Sdk="Xamarin.Android.Sdk/0.0.1"` right now, but we shouldn't document that.

Reply:
I think the project SDK will also be `Microsoft.NET.Sdk`, and our SDK will be imported from there ultimately. Not sure if we should make this distinction now though.